artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -112,6 +112,11 @@ if [ -z "$WILDFLY_COMMON" ] ; then
   WILDFLY_COMMON="$ARTEMIS_HOME/lib/${wildfly-common}"
 fi
 
+if [ -f "$ARTEMIS_INSTANCE/log/oomheapdump.hprof" ] ; then
```

Comment:
Rem this, also best to use a var for the filename as everyone will have their own naming conventions
 Lastly you may want to have a max history else you can easily fill disk

Reply:
E.g. we where i am atm all heap dumps include the process id, service name (as multiple instances on a host) and timestamp.

Reply:
I'll add var for the filename but I set a fixed filename to prevent the creation of multiple heap dumps that can fill the disk.
I added this check because if the file exists, no other dumps are created. The mv command replaces the previous backup so there will be at most 2 dumps.